src/datatype/__tests__/mixed.test.ts

```diff
@@ -1,104 +1,107 @@
-// @ts-ignore TS6133
-import { expect, test } from "@jest/globals";
-import { string, mixed } from "../";
+import { mixed, MixedType, string, StringType, number } from '../';
+import { ErrorSet, InvalidTypeError, InvalidFieldError, TooBigError } from '../../error';
 
-import { ErrorSet, InvalidTypeError } from "../../error";
-
-describe("mixed type", () => {
-  const customer = mixed({
-    name: string(),
-  });
-
-  test("valid type", () => {
-    const pass = customer.parser({
-      name: "Gialynguyen",
+describe("DataType Mixed", () => {
+    const subject = mixed({
+        name: string(),
+        age: number(),
     });
 
-    expect(pass).toEqual({
-      name: "Gialynguyen",
+    it('should have instance of MixedType', () => {
+        const mixedValue = { name: 'Người Da Màu', age: 24 };
+        expect(subject).toBeInstanceOf(MixedType);
+        expect(subject.children['name']).toBeInstanceOf(StringType);
+        expect(subject.parser(mixedValue)).toEqual(mixedValue);
+        expect(subject.parser({ age: 24 }, { tryParser: true, paths: [] }).name).toBeUndefined();
+        expect(subject.parser({ age: 20 }, { deepTryParser: true, tryParser: true, paths: [] }).name).toBeUndefined();
```

Comment:
should add a nested property in order to test the `deepTryParser` case.

Reply:
I've updated this case. Please take a look.